examples/responses/response_cookies_4.py

```diff
@@ -0,0 +1,40 @@
+from random import randint
+
+from pydantic import BaseModel
+
+from starlite import Response, Router, Starlite, get
+from starlite.datastructures import Cookie
+
+
+class Resource(BaseModel):
+    id: int
+    name: str
+
+
+@get("/resources")
+def retrieve_resource() -> Resource:
+    return Resource(
+        id=1,
+        name="my resource",
+    )
+
+
+def after_request_handler(response: Response) -> Response:
+    response.set_cookie(**Cookie(key="Random-Cookie", value=str(randint(1, 100))).dict)
+    return response
+
+
+router = Router(
+    path="/router-path",
+    route_handlers=[retrieve_resource],
+    after_request=after_request_handler,
+    response_cookies=[
```

Comment:
@provinzkraut could you please check. It looks like a real bug.

Adding `response_cookies` param to the router makes Starlite to emit `set-cookie` header twice. In this case second header has empty value.

Reply:
I `xfaile`ed this test for now.